package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dat",
-  "version": "13.7.0",
+  "version": "13.8.0",
```

Comment:
I'm gonna try to get https://github.com/datproject/dat/pull/828 released for 13.8 too, can we increment version separate from the PR?